docs/source/guide/dashboard_annotator.md

```diff
@@ -24,14 +27,23 @@ This can be useful in a number of ways:
 
 ## Access the dashboard
 
-The dashboard is available from the Organization page, meaning that your user role must be an Owner or Administrator to have the necessary permissions to view it. 
+| User role             | Access restrictions                                                                                  |
+|-----------------------|------------------------------------------------------------------------------------------------------|
+| Owner <br><br> Admin      | Can access from the Organization and the Home page. <br> Can filter for all users. <br> Can filter for all workspaces. |
+| Manager               | Can access from the Home page. <br> Can filter for all users. <br> Can only filter for workspaces or projects in which they are a member. |
+| Reviewer <br><br> Annotator | Can access from the Home page. <br> Can only see their own annotation history. <br> Can only filter for workspaces or projects in which they are a member. |
 
-From the organization members list, select the user you want to view. Annotator performance reports are available for users in all roles, not just the Annotator role. 
+There are two ways to access the dashboard: 
 
-With the user selected, click **Annotator Performance Report** on the right. 
+* From the **Organization** page, select a user on the **Members** page. Then select **Annotator Performance Report** on the right. 
+* From the Home page, click **Annotator Dashboard**. 
```

Comment:
There is actually a third way.
you can access the APD from the Project members page by clicking on the vertical ellipsis next to one of the users and clicking `View Performance Page`
<img width="494" height="264" alt="image" src="https://github.com/user-attachments/assets/b2aa9207-857d-4c1a-918a-1c1a79819e3a" />
it will pre-select that user and project